participants/Decentral Myvlog.md

```diff
@@ -0,0 +1,59 @@
+
+(0) Project Name and Team Slack Handles
+project's name:Decentral Myvlog
+Filecoin Slack: myvlog@decentral.cc
+
+(1) Brief Project Description
+Free and amazing app to record your vlog!
+
+(2) Link to Project UI
+Please include a link to your project application or UI.
+https://github.com/DecentralTech/myui

```

Comment:
Is there a live website for this, or just a plan to have a ui? The current UI seems to only contain images.

Reply:
@GregTheGreek , ok, we're preparing more UI pages and features for the app, you will see it very soon...